certora/harness/MTokenHarness.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity 0.8.23;
+import "../../src/MToken.sol";
+
+
+contract MTokenHarness is MToken {
+
+    constructor(address ttgRegistrar_, address minterGateway_) MToken(ttgRegistrar_, minterGateway_) {}
+
+    /******************************************************************************************************************\
+    |                                                     Getters                                                      |
+    \******************************************************************************************************************/
+
+
+    function getLatestIndexInMToken() public view returns (uint128) {
+        return latestIndex;
+    }
+
+    function getLatestRateInMToken() public view returns (uint32) {
+        return _latestRate;
+    }
+
+    function getLatestUpdateTimestampInMToken() public view returns (uint40) {
+        return latestUpdateTimestamp;
+    }
```

Comment:
Why is this needed, isn't `latestUpdateTimestamp` already a public getter? And why is there a redundant `InMToken` suffix in the function name?